babel/src/test/resources/sql/big-query.iq

```diff
@@ -1691,16 +1691,38 @@ SELECT TIMESTAMP_DIFF("2001-02-01 01:00:00", "2001-02-01 00:00:01", HOUR) AS neg
 #     corresponding Gregorian calendar year.
 #
 # Returns DATE
-
-!if (false) {
-SELECT DATE_TRUNC(DATE '2008-12-25', MONTH) AS month;
+WITH Dates AS (
+  SELECT DATE_TRUNC(DATE '2008-12-25', YEAR) as d UNION ALL
```

Comment:
can you add an extra column, say `'year' as frame, ` in row 1, 'isoyear' in row 2, just in case the rows get reordered.